documentation/src/docs/admin-guide/lts-sizing.md

```diff
@@ -106,4 +110,47 @@ the thread pool size to 2-4 threads.
 - _Maximum size of single chunk in LTS: In production, it may be required to revisit the value of `writer.rollover.size.bytes.max`.
 Limiting the size of chunks in LTS poses a trade-off: if the value is low, the system will achieve fine retention granularity 
 (good) but extra metadata overhead (not so good). On the contrary, a high value in this parameter induces coarse retention 
-granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios. 
\ No newline at end of file
+granularity with less metadata overhead. Limiting chunk sizes between 1GB-16GB seems reasonable in many scenarios.
+
+## Right-Sizing Tier-1 Storage (Bookkeeper)
+
+Pravega makes use of tier-1 storage to store durably and with low latency events until they are moved to long-term storage. 
+Thus, we need guidelines to right-sizing the storage capacity required in tier-1 (i.e., Bookkeeper).
+
+To this end, there are three variables that we need to take into account:
+- _Max ledger size (`bookkeeper.ledger.size.max`)_: Maximum Ledger size (bytes) in BookKeeper. Once a Ledger reaches this size, 
+it will be closed and another one open. _Type_: `Long`. _Default_: `1073741824`. _Update-mode_: `per-server`.
+
+- _Number of Segment Containers (`pravegaservice.container.count`, `controller.container.count`)_: Number of Segment Containers in the system. 
+This value must be the same across all Segment Store instances in this cluster. This value cannot be changed dynamically. See documentation for details.
```

Comment:
'See documentation for details.' -- Which docs? Can we link to them?

Reply:
The description of this parameter is the same as in the `config.properties` file: https://github.com/pravega/pravega/blob/e05a7fb8275a97a4c2962bd5d2a18a02f539b2f4/config/config.properties#L41
We don't have links on `confg.properties` neither. Maybe we could add a general link like this one: https://cncf.pravega.io/docs/nightly/segment-store-service/#segment-containers or maybe this other document that also mentions that: https://github.com/pravega/pravega/blob/master/documentation/src/docs/segment-containers.md#segment-containers